cypress/e2e/CMS/$BlogSeries.feature

```diff
@@ -0,0 +1,196 @@
+        And user fills out the following fields
+            | fieldLabel | value                      | field_name       |
+            | Pretty URL | blog-post-test-blog-series | field_pretty_url |
+        And browser waits
+        And user fills out the following fields
+            | fieldLabel               | value                                            | field_name                     |
+            | Page Title               | Automated Test Blog Post for testing Blog Series | title                          |
+            | Browser Title            | Automated Test Blog Post - Browser Title         | field_browser_title            |
+            | Card Title               | Automated Test Blog Post - Card Title            | field_card_title               |
+            | Meta Description         | Automated Blog Post Meta Description             | field_page_description         |
+            | Feature Card Description | Automated Test Blog Post - Feature Card Desc     | field_feature_card_description |
+            | Author                   | Automated Test Blog Post Author                  | field_author                   |
+        And user enters "Blog Post List Description" into "List Description" text field
+        And user enters "Blog Post Intro Text" as intro text
+        And user fills out "Body" text area with "This is a description of Blog Post content type."
+        And user selects "Published" from "Save as" dropdown
+        Then user saves the content page
+
+    Scenario: Edit and republish blog series content type
+        Given user is navigating to "/user/login"
+        When user enters credentials
+        And user clicks "Log in" button
+        Then user is logged in and the user name "admin" is displayed in the toolbar
+        And the tool bar appears at the top
+        When user clicks on "Content" tab
+        And user clicks on title with url "blog-series" from the list of content
+        And user clicks on the tool bar status green button "Published"
+        And user clicks "View in edit form" button from other actions
+        And user clears out "Pretty URL" field
+        And user clears out "Page Title" field
+        And user clears out "Meta Description" field
+        And user fills out the following fields
+            | fieldLabel       | value                                              | field_name             |
+            | Pretty URL       | blog-series-edited                                 | field_pretty_url       |
+            | Page Title       | Automated Test Blog Series Edited                  | title                  |
+            | Meta Description | Automated Test Blog Series Meta Description Edited | field_page_description |
+        And user clicks on dropdown button toggle to view Featured Posts
+        And user clicks on "Select entities" button to link to the Featured Post
+        Then "Select entities" section appears
+        And browser waits
```

Comment:
I see a bunch of browser waits here ( counted 9 just in this scenario, which is equal to 18 seconds of added wait) 

What is the reason for all those waits?  and can we instead of pausing for 2 secs, maybe add a step to verify that some expected element exists ?

Reply:
done